src/concrete/ml/sklearn/base.py

```diff
@@ -1280,8 +1280,31 @@ def __init__(self, n_bits: int):
         #: The model's inference function. Is None if the model is not fitted.
         self._tree_inference: Optional[Callable] = None
 
+        #: Boolean that indicates whether the rounding feature is enabled or not.
+        self.__use_rounding: bool = True
+
         BaseEstimator.__init__(self)
 
+    def use_rounding(self, value: bool) -> None:
```

Comment:
as Andrei suggested, we can maybe rename this like `use_optimized_execution`, or something similar !